runtime/common/src/slots.rs

```diff
@@ -427,20 +431,28 @@ impl<T: Config> Leaser for Pallet<T> {
 			.unwrap_or_else(Zero::zero)
 	}
 
-	fn lease_period() -> Self::LeasePeriod {
-		T::LeasePeriod::get()
+	#[cfg(any(feature = "runtime-benchmarks", test))]
+	fn lease_period_length() -> (T::BlockNumber, T::BlockNumber) {
+		(T::LeasePeriod::get(), T::LeaseOffset::get())
```

Comment:
This API is only needed for benchmarks, so we hide it from the public API, and therefore eliminate any chance that a user may try to calculate this on their own.